models/discounted-price/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@commercetools-test-data/discounted-price",
+  "version": "10.1.3",
+  "description": "Data model for commercetools API DiscountedPrice",
+  "bugs": "https://github.com/commercetools/test-data/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/test-data.git",
+    "directory": "models/discounted-price"
+  },
+  "keywords": [
+    "javascript",
+    "typescript",
+    "test-data"
+  ],
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/commercetools-test-data-discounted-price.cjs.js",
+  "module": "dist/commercetools-test-data-discounted-price.esm.js",
+  "files": [
+    "dist",
+    "package.json",
+    "LICENSE",
+    "README.md"
+  ],
+  "dependencies": {
+    "@babel/runtime": "^7.17.9",
+    "@babel/runtime-corejs3": "^7.17.9",
+    "@commercetools-test-data/commons": "10.1.3",
+    "@commercetools-test-data/core": "workspace:10.8.1",
```

Comment:
We're missing the `@commercetools-test-data/product-discount` package as we need it because is a nested model in the Graphql version of this new model.